modules/processing/network.py

```diff
@@ -111,6 +111,21 @@ def __init__(self, filepath):
         self.irc_requests = []
         # Dictionary containing all the results of this processing.
         self.results = {}
+        # List containing all whitelist entries.
+        self.whitelist = self._build_whitelist()
+
+    def _build_whitelist(self):
+        result = []
+        whitelist_path = os.path.join(
+            CUCKOO_ROOT, "data", "whitelist", "domain.txt"
+        )
+        f = open(whitelist_path, 'rb')
```

Comment:
i think it would be more better/pretty
 change this
```
+        f = open(whitelist_path, 'rb')
 +        while True:
 +            line = f.readline()
 +            if not line: break
 +            result.append(line.split('\n')[0])
 +        
```

to this

```
f = open(whitelist_path, 'rb').readlines()
f = [whitelisted.strip() for whitelisted in f]
```